presto-docs/src/main/sphinx/functions/aggregate.rst

```diff
@@ -572,7 +573,7 @@ That is, for a random variable :math:`x`, they approximate
 
 .. math ::
 
-    H(x) = - \int x \log_2\left(f(x)\right) dx,
+    h(x) = - \int x \log_2\left(f(x)\right) dx,
```

Comment:
why? Also, put documentation changes into their own commit

Reply:
#13163 is the first diff which combines functions of discrete entropy with those of differential entropy. Differential entropy is more precisely indicated by an $h$, not $H$, but that is sometimes ignored when on its own. When combining both discrete and differential entropy (as the docs of #13163 do or will do), it's important to differentiate between the two.